Gemfile.lock

```diff
@@ -283,10 +316,17 @@ GEM
     zeitwerk (2.7.1)
 
 PLATFORMS
-  arm64-darwin-22
-  arm64-darwin-23
-  x86_64-darwin-21
-  x86_64-linux
+  aarch64-linux-gnu
```

Comment:
Do we need these other platforms if we are using ruby one now?

Reply:
No. I probably forgot to delete the Gemfile.lock. You need to do that to remove other platforms.